bucket/scriptcs.json

```diff
@@ -7,10 +7,6 @@
     "extract_dir": "tools",
     "bin": "scriptcs.exe",
     "checkver": {
-        "url": "https://chocolatey.org/packages/ScriptCs",
-        "regex": "ScriptCs ([\\d.]+)"
-    },
-    "autoupdate": {
-        "url": "https://packages.chocolatey.org/ScriptCs.$version.nupkg"
+        "choco": "ScriptCs"
```

Comment:
I think this line should be in `checkver`.

Reply:
Since sometimes Chocolatey page name is not the same as `$app`...

I'll add the option to use just `"checkver": "choco"`, just like `"checkver": "github"`.